contracts/libraries/OriginationLibrary.sol

```diff
@@ -41,6 +41,11 @@ library OriginationLibrary {
         RolloverAmounts migrationAmounts;
     }
 
+    struct SwapParameters {
```

Comment:
This can be part of `ICrossCurrencyRollover`, since it is only used in the cross-currency case